spec/Pim/Bundle/CustomEntityBundle/Checker/ProductLinkCheckerSpec.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace spec\Pim\Bundle\CustomEntityBundle\Checker;
+
+use Doctrine\ORM\EntityManagerInterface;
+use PhpSpec\ObjectBehavior;
+use Pim\Bundle\CustomEntityBundle\Checker\ProductLinkChecker;
+use Pim\Bundle\CustomEntityBundle\Checker\ProductLinkCheckerInterface;
+use Pim\Bundle\CustomEntityBundle\Repository\AttributeRepository;
+use Pim\Component\Catalog\Query\ProductQueryBuilderFactoryInterface;
+
+class ProductLinkCheckerSpec extends ObjectBehavior
+{
+    public function let(
```

Comment:
You should remove the method's visibility, see https://github.com/phpspec/phpspec/pull/590#issuecomment-67995169